### PR TITLE
[OSDOCS-12387]: Updating ARN text for HCP docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-12387

Link to docs preview: https://83843--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-aws.html#hcp-aws-create-role-sts-creds_hcp-deploy-aws

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-aws-create-role-sts-creds.adoc [hunk 1]
```diff
@@ -17,7 +17,13 @@ Before creating a hosted cluster on {aws-first}, you must create an {aws-short}
 $ aws sts get-caller-identity --query "Arn"
 ----
 +
-Note the ARN of your user to use in the next step.
+.Example output
+[source,terminal]
+----
+arn:aws:iam::1234567890:user/<aws_username>
```
Comment (heliubj18) [inline modules/hcp-aws-create-role-sts-creds.adoc:23]: if we just want to get the arn without double quotations, I think the command should be 
`$ aws sts get-caller-identity --query "Arn" --output text`

e.g.
$ aws sts get-caller-identity --query "Arn"
"arn:aws:iam::***:user/heli"
$ aws sts get-caller-identity --query "Arn" --output text
arn:aws:iam::***:user/heli

Comment (lahinson) [reply modules/hcp-aws-create-role-sts-creds.adoc:23]: Thanks! I added `--output text`.

File: modules/hcp-aws-create-role-sts-creds.adoc [hunk 2]
```diff
@@ -14,10 +14,16 @@ Before creating a hosted cluster on {aws-first}, you must create an {aws-short}
...
+----
++
+You need the `arn` value from the output to use in the next step.
```
Comment (bergerhoffer) [inline modules/hcp-aws-create-role-sts-creds.adoc:26]: The way this is written, I'm not sure if the "arn" value is just a piece of this output, or if it's the entire output, inserted into the <arn> replaceable value in the next step.

If it's the former then clarifying which piece to use would be helpful.

If it's the latter maybe something like the following would be clearer?

```suggestion
Use this output as the value for `<arn>` in the next step.
```

**Review by bergerhoffer — COMMENTED** [review]
A suggestion!

File: modules/hcp-aws-create-role-sts-creds.adoc (hunk 2 above)
Comment (lahinson) [reply modules/hcp-aws-create-role-sts-creds.adoc:26]: Good suggestion!

**Pull request merged**